*** GammaProto.ts	2021-10-29 23:43:20.000000000 -0700
--- GammaProto-patched.ts	2021-10-29 23:43:19.000000000 -0700
*************** export class TerImageSeq extends pb_1.Me
*** 2146,2155 ****
                      reader.readMessage(message.tmag, () => pb_1.Message.addToRepeatedWrapperField(message, 2, TerImage.deserialize(reader), TerImage));
                      break;
                  case 3:
!                     pb_1.Message.addToRepeatedField(message, 3, reader.readPackedInt32());
                      break;
                  case 4:
!                     pb_1.Message.addToRepeatedField(message, 4, reader.readPackedInt32());
                      break;
                  default: reader.skipField();
              }
--- 2146,2155 ----
                      reader.readMessage(message.tmag, () => pb_1.Message.addToRepeatedWrapperField(message, 2, TerImage.deserialize(reader), TerImage));
                      break;
                  case 3:
!                     message.zval = reader.readPackedInt32();
                      break;
                  case 4:
!                     message.indx = reader.readPackedInt32();
                      break;
                  default: reader.skipField();
              }
